conf/bookkeeper.conf

```diff
@@ -268,6 +268,9 @@ serverTcpNoDelay=true
 # The Recv ByteBuf allocator max buf size.
 # byteBufAllocatorSizeMax=1048576
 
+# The maximum netty frame size in bytes. Any message received larger than this will be rejeted. Default value is 1G.
```

Comment:
```suggestion
# The maximum netty frame size in bytes. Any message received larger than this will be rejected. The default value is 1G.
```

Reply:
@liudezhi2098 It's better to use 5MB as the default value, in order to avoid the change of the default value to break the use of some scenes.

Reply:
use 5MB as the default value , will cause problems，can be changed to 5253120（
5242880 + Commands.MESSAGE_SIZE_FRAME_PADDING）